inji-web/src/__tests__/components/DataShare/CustomTimes/CustomExpiryTimesContent.test.tsx

```diff
@@ -0,0 +1,76 @@
+import {fireEvent, render, screen} from "@testing-library/react";
+import {CustomExpiryTimesContent} from "../../../../components/DataShare/CustomExpiryTimes/CustomExpiryTimesContent";
+import {useTranslation} from "react-i18next";
+
+describe("Test the Layout of the Custom Expiry Content", () => {
+
+    beforeEach(() => {
+        jest.mock("react-i18next", () => ({
+            useTranslation: () => ({
+                t: (key: string) => key,
+            }),
+        }));
+    } )
+
+    test("Test the presence of the Outer Container", ()=>{
+        const expiryMockFn = jest.fn();
+        render(<CustomExpiryTimesContent expiryTime={1} setExpiryTime={expiryMockFn} />);
+        const ctDocument = screen.getByTestId("CustomExpiryTimesContent-Outer-Container");
+        expect(ctDocument).toBeInTheDocument();
+    })
+    test("Test the presence of the Time Range Container", ()=>{
+        const expiryMockFn = jest.fn();
+        render(<CustomExpiryTimesContent expiryTime={1} setExpiryTime={expiryMockFn} />);
+        const ctDocument = screen.getByTestId("CustomExpiryTimesContent-Times-Range-Container");
+        expect(ctDocument).toBeInTheDocument();
+    })
+    test("Test the presence of the Time Range Increase Container", ()=>{
+        const expiryMockFn = jest.fn();
+        render(<CustomExpiryTimesContent expiryTime={1} setExpiryTime={expiryMockFn} />);
+        const ctDocument = screen.getByTestId("CustomExpiryTimesContent-Times-Range-Increase");
+        expect(ctDocument).toBeInTheDocument();
+    })
+    test("Test the presence of the Time Range Decrease Container", ()=>{
+        const expiryMockFn = jest.fn();
+        render(<CustomExpiryTimesContent expiryTime={1} setExpiryTime={expiryMockFn} />);
+        const ctDocument = screen.getByTestId("CustomExpiryTimesContent-Times-Range-Decrease");
+        expect(ctDocument).toBeInTheDocument();
+    })
+    test("Test the presence of the Time Range Value", ()=>{
+        const expiryMockFn = jest.fn();
+        render(<CustomExpiryTimesContent expiryTime={1} setExpiryTime={expiryMockFn} />);
+        const ctDocument = screen.getByTestId("CustomExpiryTimesContent-Times-Value");
+        expect(ctDocument).toBeInTheDocument();
+    })
+    test("Test the presence of the Time Range Metrics", ()=>{
+        const expiryMockFn = jest.fn();
+        render(<CustomExpiryTimesContent expiryTime={1} setExpiryTime={expiryMockFn} />);
+        const ctDocument = screen.getByTestId("CustomExpiryTimesContent-Times-Metrics");
+        expect(ctDocument).toBeInTheDocument();
+        expect(ctDocument).toHaveTextContent("metrics");
+    })
```

Comment:
These tests can be replaced by simple snapshot tests which will help to test layout with styling

Reply:
Snapshot Test we will take it up as part of different card mentioned.